### DISCUSSION TITLE
Reverse Geocoding

### DISCUSSION THREAD

**Discussion by PranshulGG** [bmaroti9/Overmorrow#43] [Q&A] [answered]
Does the app use reverse geocoding? If yes, what API or service are you using?

**Comment by bmaroti9** [comment, marked as answer]
Hi! 
So i originally used the weatherapi reverse geocoding, but it didn't work totally consistently, and i was getting reviews of people getting placed 5 km away from their actual city, so I switched to this flutter plugin that somehow does it for you https://pub.dev/packages/geocoding. You could look around in the code and the readme, supposedly it uses free geocoding services.
